src/OSPSuite.Core/Services/IDialogCreator.cs

```diff
@@ -24,8 +26,12 @@ public interface IDialogCreator
       /// </remarks>
       ViewResult MessageBoxYesNoCancel(string message, string yes, string no, string cancel);
 
+      ViewResult MessageBoxYesNoCancel(string message, string yes, string no, string cancel, ViewResult defaultButton);
+
       ViewResult MessageBoxYesNo(string message);
 
+      ViewResult MessageBoxYesNo(string message, ViewResult defaultButton);
```

Comment:
we did not have a way to specify a default button. I have changed the implementation to support a default button that is not yes